test/e2e/datafederation_pe_test.go

```diff
@@ -54,175 +64,63 @@ var _ = Describe("UserLogin", Label("datafederation"), func() {
 		})
 	})
 
-	DescribeTable("Namespaced operators working only with its own namespace with different configuration",
-		func(test *model.TestDataProvider, pe []privateEndpoint) {
-			testData = test
-			actions.ProjectCreationFlow(test)
-			dataFederationFlow(test, providerAction, pe)
-		},
-		Entry("Data Federation can be created with private endpoints", Label("datafederation-pe-aws"),
-			model.DataProvider(
-				"privatelink-aws-1",
-				model.NewEmptyAtlasKeyType().UseDefaultFullAccess(),
-				40000,
-				[]func(*model.TestDataProvider){},
-			).WithProject(data.DefaultProject()),
-			[]privateEndpoint{
-				{
-					provider: "AWS",
-					region:   config.AWSRegionEU,
+	It("Creates a data federation with private endpoint", func(ctx context.Context) {
+		var pe *cloud.PrivateEndpointDetails
+		const dataFederationInstanceName = "test-data-federation-aws"
+
+		//nolint:dupl
+		By("Create privaye endpoint in AWS", func() {
```

Comment:
Typo (probably not worth rerunning tests for)

Reply:
ah, good catch 🙈